clang/include/clang/Basic/DiagnosticCommonKinds.td

```diff
@@ -356,6 +356,8 @@ def err_file_modified : Error<
   "file '%0' modified since it was first processed">, DefaultFatal;
 def err_file_too_large : Error<
   "sorry, unsupported: file '%0' is too large for Clang to process">;
+def err_expansions_too_large : Error<
```

Comment:
I also don't feel strongly there. The reason I introduced a new warning is because this could theoretically happen without any `#include` and I did not want to touch the existing error message.

Reply:
SGTM (but the current version is also OK with me)

Reply:
I have reused the existing diagnostic and updated it to remove a mention of include.